tests/python/frontend/tflite/test_forward.py

```diff
@@ -843,6 +843,13 @@ def _test_pow(data):
     """ One iteration of power """
     return _test_elemwise(math_ops.pow, data)
 #######################################################################
+# Squared_difference
+# ------------------
+
+def _test_squared_difference(data):
```

Comment:
Same problem as previous comment. Please correct it.

Reply:
Done now